SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -31,8 +31,8 @@ namespace
 constexpr float FINISH_TEST_GRACE_PERIOD_DURATION = 2.0f;
 } // namespace
 
-ASpatialFunctionalTest::ASpatialFunctionalTest()
-	: Super()
```

Comment:
I think the default constructor is fine, so we should keep it to avoid changing the existing GDK tests. The following is an extract from https://docs.unrealengine.com/en-US/API/Runtime/CoreUObject/UObject/UObject/index.html
```

UObject() | Default constructor
-- | --
  | UObject(    const FObjectInitializer & ObjectIn...) | Deprecated constructor, **ObjectInitializer is no longer needed but is supported for older classes.**
  | UObject(    FVTableHelper & Helper) | DO NOT USE. This constructor is for internal usage only for hot-reload purposes.
  | UObject(    EStaticConstructor ,    EObjectFlags InFlags) | DO NOT USE. This constructor is for internal usage only for statically-created objects.
```

Reply:
Thanks for the review! I never found that, I primarily looked at the docs for FVTable/FObjectinitializer and the FVTable just said never use these and that was what was throwing an error when I was getting it to build when I started the task. Never saw the CoreObject note! I'll close the PR and see if I can get the effect working from the other way! Thanks Victoria!